api/web3server.go

```diff
@@ -725,6 +726,34 @@ func (svr *Web3Server) getTransactionByBlockNumberAndIndex(in interface{}) (inte
 	return svr.getTransactionCreateFromActionInfo(actionInfos[0])
 }
 
+func (svr *Web3Server) getStorageAt(in interface{}) (interface{}, error) {
+	ethAddr, err := getStringFromArray(in, 0)
+	if err != nil {
+		return nil, err
+	}
+	storagePos, err := getStringFromArray(in, 1)
+	if err != nil {
+		return nil, err
+	}
+	ioAddr, err := ethAddrToIoAddr(ethAddr)
+	if err != nil {
+		return nil, err
+	}
+	contractAddr, err := address.FromString(ioAddr)
+	if err != nil {
+		return nil, err
+	}
+	pos, err := hex.DecodeString(removeHexPrefix(storagePos))
+	if err != nil {
+		return nil, err
+	}
+	val, err := svr.coreService.ReadContractStorage(context.Background(), contractAddr, pos)
+	if err != nil {
+		return nil, err
+	}
+	return "0x" + hex.EncodeToString(val), nil
+}
```

Comment:
can you add a test?
1. deployContractV2()
2. read this contract pos = 0x0, it should return 32-byte all 0